src/tests/block-tests.ts

```diff
@@ -630,4 +630,111 @@ describe('block tests', () => {
     expect(blockQuery.body.execution_cost_write_count).toBe(3);
     expect(blockQuery.body.execution_cost_write_length).toBe(3);
   });
+
+  test('blocks v2 filtered by burn block', async () => {
```

Comment:
Should there be a test for querying a burn block that has multiple stacks blocks?

Reply:
This test creates a few Stacks blocks per Bitcoin block and then filters by burn block hash to check for result count, etc. Is that what you mean? Or you mean calling the `/burn_block` endpoint also in this test?